docs/UserGuide.md

```diff
@@ -400,14 +418,15 @@ Examples:
 
 #### Listing all tags: `list`
 
-Lists all tags in the remarks panel. It lists each tag together with the contacts tagged. Super-tags are also denoted with `(supertag)`.
+Lists all tags in the remarks panel including the corresponding contacts of each tag. 
+Super-tags are also denoted with `(supertag)`.
```

Comment:
if we split this into two separate points 'also' is unnecessary in the second sentence.

Reply:
In the context of markdown, the change does not make any difference.

Reply:
I did not edit the second sentence, as the only edit I made with it was to move it to the next line.